### PR TITLE
Add trailing blank character feature

### PR DESCRIPTION
**Pull request by FrBrGeorge**
I've try to use pyfiglet to generate _pixel-like_ data and failed a bit, because:

1. All lines generated by pyfiglet have different lengths
1. Sometimes spaces are stripped off (especially trailing ones)

So I've implemented a couple of simple new options

### CODE REVIEW

File: pyfiglet/__init__.py [hunk 1]
```diff
@@ -906,6 +917,10 @@ def main():
     parser.add_option('-w', '--width', type='int', default=80, metavar='COLS',
                       help='set terminal width for wrapping/justification '
                            '(default: %default)')
+    parser.add_option('-b', '--blank', type='str', default=' ', metavar='BLANK',
+                      help='set blank character (default: "%default")')
+    parser.add_option('-e', '--exact', action='store_true', default=False,
+                      help='add additional blank characters to keep exact width')
```
Comment (pwaller) [inline pyfiglet/__init__.py:923]: What do you think to calling `--exact` `--add-trailing-space` instead? `--exact` isn't very meaningful for me, but perhaps you and I can come up with something which would make it more obvious.

Comment (FrBrGeorge) [reply pyfiglet/__init__.py:923]: Anything you want, I'm more interested in functionality than in syntax :)

Comment (FrBrGeorge) [reply pyfiglet/__init__.py:923]: `-add-trailing-space` is obvious enough, anyway

Comment (pwaller) [reply pyfiglet/__init__.py:923]: Sorry I didn't respond in a while. How about `--trailing-space` with short option `-T`? If you make the change, I'll be likely to accept. I'm also likely to accept other spellings if you have something better than `--exact`.

**Review by pwaller — CHANGES_REQUESTED** [review]

**Comment by pwaller** [comment]
Ping @FrBrGeorge for response to review comments.

File: pyfiglet/__init__.py (hunk 1 above)
Comment (FrBrGeorge) [reply pyfiglet/__init__.py:940]: Ahh, this was the point when I've lost the point ☹. So sorry for such a delay — didn't have resources to sort this out.

To be clear: you want me to rename _both_ of options, or just `--exact` one? For me `-b/--blank` looks descriptive enough — this is the character we will use __in place of blank__.

The meaning of `--excact` is «all lines of banner should be exact one width». I agree that «exact» alone tells nothing, but I believe it's important to keep direct «equal width» meaning, not just «add trailing characters» (also: they're not just spaces anymore☺). How about «`--equal-wdth`  / `-e`»?

But if you vote for `--trailing-space`/`-T` — it's OK, I'll accept that ☺

So what can I do to accelerate this?

**Comment by pwaller** [comment]
Hi, Ping in case you intend to still follow up with this, if not please close. (Also please in the future ping me if you're waiting on me)

File: pyfiglet/__init__.py (hunk 1 above)
Comment (pwaller) [reply pyfiglet/__init__.py:940]: Apologies, I somehow missed this response before. As you can see, I come back to this project every once in a rare while. And apologies for holding it up with naming bike-shedding. Let's try and complete this.

Thinking about for a few minutes and coming back to it, I would accept:

* `--exact-width` ('exact' on its own could mean something else, adding the word 'width' is more descriptive)
* `--extra-padding`
* `--trailing-space`
* More suggestions.

I should have a small amount of time to action this in the coming Fridays, so feel free to ping the thread on a weekly basis if you don't hear back, and if I don't hear back, I'll make the adjustments at some point in the future and include the feature.

**Comment by pwaller** [comment]
Closing as idle. We can reopen this if there is still an interest it in from anyone.

**Pull request closed without merging**